packages/features/bookings/lib/handleNewBooking/test/fresh-booking.test.ts

```diff
@@ -161,6 +161,7 @@ describe("handleNewBooking", () => {
         });
 
         const createdBooking = await handleNewBooking(req);
+        const iCalUID = getICalUID({ uid: createdBooking.uid });
```

Comment:
I don't like the idea of using `getICalUID` in the test. It is the part of code being tested, so it shouldn't be used in test ideally. It could cause a situation like 'not-working' === 'not-working' is true so it's working.

Because we are controlling the scenario here, we should be knowing exactly what id was chosen to be iCalUID among `event.iCalUID`, `event.uid`, `createdBooking.uid`, or a brand new ID. So, we should use that value here directly to verify.

Reply:
Fixed it cc @joeauyeung